dashboards-reports/server/config/schema.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { schema, TypeOf } from '@osd/config-schema';
+
+const OsdServerSchema = schema.object({
+  hostname: schema.maybe(
+    schema.string({
+      validate(value) {
+        if (value === '0') {
+          return 'must not be "0" for the headless browser to correctly resolve the host';
+        }
+      },
+      hostname: true,
```

Comment:
what does this do?

Reply:
validator option for `hostname`
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/e9526f5927c7224226374b9d49c272c50c3122c6/packages/kbn-config-schema/src/types/string_type.ts#L37